ext/README.md

```diff
@@ -0,0 +1,12 @@
+# ext folder for projects using kubernetes playgound
+
+External projects can create their own folder here.
+The files and folders under `ext` are not visible by git.
+Taking the kites project as an example, this is the recommended
+folder structure:
```

Comment:
directory

Reply:
I've changed into "The files and directory under `ext` are not synced with this git
repository."  This is the point that I'd like to make... normally a directory inside a git repository is synced with the repository itself... here we have a directory that can be used by the project without interfering with the repo